data-tck/src/main/java/io/micronaut/data/tck/repositories/BookSpecifications.java

```diff
@@ -0,0 +1,15 @@
+package io.micronaut.data.tck.repositories;
+
+import io.micronaut.data.model.jpa.criteria.PersistentEntityFrom;
+import io.micronaut.data.repository.jpa.criteria.PredicateSpecification;
+import io.micronaut.data.tck.entities.Book;
+
+public class BookSpecifications {
+
+    public static PredicateSpecification<Book> titleEquals(String title) {
```

Comment:
Please also test that it will fetch the result if you only do the join without the predicate:
```java
    public static PredicateSpecification<Book> justJoin() {
        return (root, criteriaBuilder) ->  {
            ((PersistentEntityFrom) root).join("genre", io.micronaut.data.annotation.Join.Type.LEFT_FETCH);
            return null;
        };
```

Reply:
Done